tests/travis/install-mysql-8.0.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo "Installing MySQL 8.0..."
+
+echo mysql-apt-config mysql-apt-config/select-server select mysql-8.0 | sudo debconf-set-selections
+wget https://dev.mysql.com/get/mysql-apt-config_0.8.10-1_all.deb
```

Comment:
Please use Docker container instead of arbitrary 3rd-party debfile. As done in #3183.

Reply:
I'm not familiar with Docker, however for the record this isn't an arbitrary debfile, this is MySQL's official APT repository.

Reply:
Given the nature of APT repositories, it's not a good idea to depend on a debfile as debfiles may randomly disappear when they get superseded by newer version.
Also Docker is easier to setup, configure environmen´t, maintain and in the end is more reliable.

Reply:
Note that MySQL 5.7 [is already installed using MySQL's APT repo](https://github.com/doctrine/dbal/blob/master/tests/travis/install-mysql-5.7.sh). If you move to Docker, then this one should be updated as well for consistency.

Reply:
Sure.